### PR TITLE
Real lbryum

### PR DESCRIPTION
**Pull request by jackrobison**

### CODE REVIEW

File: lbrynet/lbrynet_daemon/LBRYDaemon.py [hunk 1]
```diff
@@ -918,61 +918,61 @@ def create_session(results):
...
-
-        return d
-
-    def _show_first_run_result(self, credits_received):
-        if credits_received != 0.0:
-            points_string = locale.format_string("%.2f LBC", (round(credits_received, 2),), grouping=True)
-            self.startup_message = "Thank you for testing the alpha version of LBRY! You have been given %s for free because we love you. Please hang on for a few minutes for the next block to be mined. When you refresh this page and see your credits you're ready to go!." % points_string
-        else:
-            self.startup_message = None
+    # def _check_first_run(self):
```
Comment (jackrobison) [inline lbrynet/lbrynet_daemon/LBRYDaemon.py]: @alexliebowitz should this be reenabled in the daemon or should the credit request be kept in the ui?

File: lbrynet/lbrynet_daemon/LBRYDaemon.py [hunk 2]
```diff
@@ -877,13 +877,13 @@ def get_default_data_rate():
             return d
 
         def get_wallet():
-            if self.wallet_type in ["lbrycrd", "lbryum"]: #force lbrycrd wallet no matter what while lbryum is down
+            if self.wallet_type == "lbrycrd": #force lbrycrd wallet no matter what while lbryum is down
```
Comment (jobevers) [inline lbrynet/lbrynet_daemon/LBRYDaemon.py]: remove the stale comment about lbryum being down

**Comment by jobevers** [comment]
I'm terrible at it, but try to keep commits in logical groups. The `re-enable lbryum` commit also has the remove `_check_first_run` related code.  `git add -p` is useful if you've found yourself making two disjoint groups of changes.

Once you and @alexliebowitz sort out what to do with the credit request and travis passes the build - will probably need to re-run the build after merging lbryum: go SPV wallet! [LGTM](http://lgtm.in/i/2N6oxvM3q)

File: lbrynet/lbrynet_daemon/LBRYDaemonCLI.py [hunk 3]
```diff
@@ -35,7 +35,7 @@ def main():
     if len(args) > 1:
         if isinstance(args[1], dict):
             params = args[1]
-        elif isinstance(args[1], str) or isinstance(args[1], unicode):
+        elif isinstance(args[1], str, unicode):
```
Comment (jobevers) [inline lbrynet/lbrynet_daemon/LBRYDaemonCLI.py]: use `basestring`:  `isinstance(args[1], basestring)`

**Pull request merged**